README.md

```diff
@@ -7,35 +7,38 @@
 Describe the differences between a SQL and NoSQL database, and when you might use each.
 
 ```text
-Your answer...
+SQL is a relational database, NoSql is a non relational database. Sql have been the traditional databases for decades. SO Sql is the go to source typically one would use SQL for large applications that had a specific need and specific relationships. With mongo/nosql it takes a far more distributed modular approach towards getting specific data in specific way, that being said NoSQL is useful because it can incorporate both sql and nosql stuff. Nosql is often used for gaming . SQL for larger corporation databases.
 ```
 
 ### Question #2
 
 What's wrong with this Mongoose code and how might we fix it?
 
 ```js
-var results = AuthorModel.find({name: "Bob"});
+let results = AuthorModel.findOne( 'results', {name: "Bob"});
 console.log(results);
 ```
```

Comment:
The issue is that `.find` is asynchronous. In order to `console.log(results)`, we need to make sure we do that in a promise or callback attached to `.find`